src/api_test.ts

```diff
@@ -71,19 +65,16 @@ Deno.test("SlackAPI class", async (t) => {
               });
             });
 
-            await assertRejects(
```

Comment:
The API for `assertRejects` changed, leading to type errors. So, removed its use and instead used a 'manual' try/catch approach.